shellfoundry_traffic/test_helpers.py

```diff
@@ -64,23 +65,27 @@ def create_session_from_cloudshell_config():
 # 2nd generation helers.
 #
 
+
+xml_file = '/deployment.xml'
+
+
```

Comment:
Constant names should be all upper, rename DEPLOYMENT_XML.

Reply:
Changed.